Tasks/XcodeV5/task.json

```diff
@@ -239,8 +239,7 @@
             "defaultValue": "",
             "required": false,
             "helpMarkDown": "(Optional, unless you are a member of multiple development teams.) Specify the 10-character development team ID.",
-            "groupName": "sign",
-            "visibleRule": "signingOption = auto"
```

Comment:
This seems invalid in combination with "nosign" (`CODE_SIGNING_ALLOWED=NO`). I think the visibleRule should be `"visibleRule": "signingOption != nosign"`